babel.config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "presets": ["@babel/preset-react", "@babel/preset-env"],
```

Comment:
The below is largely the same, except for the removal of two babel plugins.  
The file type change was required due to updates.